GHzDACs/Cleanup/dac.py

```diff
@@ -677,8 +677,25 @@ def _setPolarity(self, chan, invert):
 
     @inlineCallbacks
     def _checkPHOF(self, op, fifoReadings, counterValue):
-        # Determine for which PHOFs (FIFO offsets) the FIFO counter equals
-        # counterValue.
+        """ Determine correct PHOF for FIFO counter of counterValue
+
+        If finds such a PHOF, sets PHOF and reads back FIFO counter to
+        verify that FIFO counter equals counterValue.
```

Comment:
"If finds..."? Is this supposed to be "If we find a valid PHOF, set PHOF and..."